projects/1.18/assets/alcocraft-beer-and-stuff/alcocraft/lang/zh_cn.json

```
@@ -0,0 +1,59 @@
+{
+  "itemGroup.alcotab": "酿酒工艺",
+
+  "block.alcocraft.spruce_keg": "酒桶",
+  "block.alcocraft.hop_block": "啤酒花",
+  "block.alcocraft.hop_plant_block": "啤酒花植株",
+
+  "item.alcocraft.hop": "啤酒花",
+  "item.alcocraft.hop_seeds": "啤酒花种子",
+  "item.alcocraft.dry_seeds": "干小麦种子",
+
+  "item.alcocraft.spruce_mug_empty": "空酒杯",
+
+  "item.alcocraft.spruce_mug_sun_pale_ale": "阳式淡色艾尔",
+  "item.alcocraft.spruce_mug_digger_bitter": "矿工苦饮",
+  "item.alcocraft.spruce_mug_nether_porter": "下界波特",

```

```suggestion
  "item.alcocraft.spruce_mug_nether_porter": "下界泊特",
```
双关的一种译法，`特`字可能译的不太好，仅供参考

---

已修改